### PR TITLE
fix: disabled state of feature flag overview page

### PR DESCRIPTION
**Pull request by JivusAyrus**
## TODO

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

image

### CODE REVIEW

File: studio/src/components/feature-flag-details.tsx [hunk 1]
```diff
@@ -64,6 +66,24 @@ const FeatureFlagOverview = ({
...
       />
     );
+  } else if (!isEnabled) {
+    content = (
+      <EmptyState
+        icon={<ExclamationTriangleIcon className="text-red-500" />}
+        title="Feature flag is not used for composition."
+        description={
+          <>
+            Feature flag is disabled. To enable the feature flag, use the below
```
Comment (df-wg) [inline studio/src/components/feature-flag-details.tsx:76]: Is there meant to be the ability for enabling the feature flag from the UI as well?

Comment (JivusAyrus) [reply studio/src/components/feature-flag-details.tsx:76]: No

Comment (JivusAyrus) [reply studio/src/components/feature-flag-details.tsx:76]: Nothing currently

**Review by df-wg — APPROVED** [review]
lgtm

**Pull request merged**